nixos-up.py

```diff
@@ -13,6 +13,9 @@
 if os.geteuid() != 0:
   sys.exit("nixos-up must be run as root!")
 
+nixos_version = os.environ.get("NIXOS_VERSION")[:5]
```

Comment:
```suggestion
nixos_version = os.environ["NIXOS_VERSION"][:5]
```
why not like this so that it raises a KeyError instead of a TypeError?

Reply:
I initially used `.get()` with a default version but removed it so this remained. I changed it :)